<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="Param">
           <attribute name="title">
            <string>Param</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="lblMemoire">
                <property name="text">
                 <string>Memoire</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QwtThermo" name="Memoire">
                <property name="maximumSize">
                 <size>
                  <width>68</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="maxValue">
                 <double>2048.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <widget class="QwtAnalogClock" name="Time">
              <property name="lineWidth">
               <number>4</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QwtKnob" name="Periode">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>118</height>
                 </size>
                </property>
                <property name="valid">
                 <bool>true</bool>
                </property>
                <property name="knobWidth">
                 <number>50</number>
                </property>
                <property name="borderWidth">
                 <number>2</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labelPerSend">
                <property name="text">
                 <string>Periode : </string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="SendPer">
                <property name="text">
                 <string>Send</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="labelPer">
                <property name="maximumSize">
                 <size>
                  <width>193</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Periode en cours : 90 minutes</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tWdgJour">
           <attribute name="title">
            <string>Jour</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QwtPlot" name="plotJour"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QCommandLinkButton" name="clBtnPvDay">
                  <property name="text">
                   <string>Prev</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCommandLinkButton" name="clBtnNxDay">
                  <property name="text">
                   <string>Next</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tWdgHisto">
           <attribute name="title">
            <string>histo</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QwtPlot" name="plotHisto">
              <zorder>Memoire</zorder>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QCheckBox" name="chBxSHT11T">
                <property name="text">
                 <string>Sht11 T</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chBxMPX">
                <property name="text">
                 <string>MPX</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chBxLM75">
                <property name="text">
                 <string>LM75</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chBxSHT11H">
                <property name="text">
                 <string>Sht11 H</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Aide</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>Outils</string>
    </property>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuOutils"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="tBarMain">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="actionClose_2">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="actionHeure_Syst_me">
   <property name="text">
    <string>Heure Syst√®me</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QwtAnalogClock</class>
   <extends>QwtDial</extends>
   <header>qwt_analog_clock.h</header>
  </customwidget>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
  <customwidget>
   <class>QwtKnob</class>
   <extends>QWidget</extends>
   <header>qwt_knob.h</header>
  </customwidget>
  <customwidget>
   <class>QwtThermo</class>
   <extends>QWidget</extends>
   <header>qwt_thermo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
